cypress/src/plugins/removeVideosForPassedTests.tsx

```diff
@@ -0,0 +1,16 @@
+import del from 'del';
```

Comment:
I think this file should have `ts` extension

Reply:
Renamed. Thanks I didn't spot it!